components/member-profile/contribution/index.js

```diff
@@ -45,43 +34,31 @@ const Contribution = ({ contribution, fullName, imageLink }) => {
     const updatedAt = +new Date(prList[0]['updatedAt']);
     if (prList[0]['state'] === 'closed') {
       completedDate = timeWas(createdAt, false, updatedAt);
-      completedText = <span>Completed in: </span>;
+      completedText = <span className={classNames.completedText}>Completed in </span>;
       featurLiveDate = timeWas(updatedAt, true);
       featureLiveOnText = `Feature live on ${featurLiveDate}`;
     }
   }
 
   return (
-    <div className={classNames.contributionContainer}>
-      <section className={classNames.heading}>
-        <span className={classNames.featureTitle}>{isTaskAvailable ? title : prList[0].title}</span>
-        <div className={classNames.prLink}>{renderPRLinks(prList)}</div>
-      </section>
-      {isTaskAvailable && (
-        <div className={showMoreContentClass}>
-          <p className={classNames.featureDescription}>{purpose}</p>
-          <p className={classNames.userIcon}>
-            <img src={imageLink} className={classNames.participantIcon} alt="participantsIcon" />
-            <span>{fullName}</span>
-          </p>
-          <p className={classNames.featureLink}>
-            <a href={featureUrl} target="_blank" rel="noreferrer">
-              Click here to checkout this feature
-            </a>
-          </p>
+    <div className={classNames.mainSection}>
+      <div className={classNames.contributionContainer}>
+        <div className={classNames.leftSection}>
+          <h3 className={classNames.featureTitle}>{isTaskAvailable ? title : prList[0].title}</h3>
```

Comment:
Nit: Can we have this logic before the JSX?
```js
const featureTitle = isTaskAvailable ? title : prList[0].title;
```

Follow up question: Will `prList` have at least one PR always?

Reply:
done